src/media/css/tiles.styl

```diff
@@ -72,11 +74,14 @@ $greater-than-mobile = unquote('(min-width: 330px)');
     }
     .preview-container {
         height: 150px;
-        margin: 0 0 -15px -10px;  // To ignore parent's padding.
+        margin: 0 0 -15px -5px;  // To ignore parent's padding.
         overflow: hidden;
         padding-bottom: 5px;
         width: 300px;
 
+        +rtl() {
+            margin-right: -10px;
```

Comment:
This is still 5px off for me, should be -15px.

![screenshot 2015-04-23 09 29 21](https://cloud.githubusercontent.com/assets/211578/7299422/50305db6-e99b-11e4-9af0-a71c718db664.png)